lib/assets/javascripts/views/devise/registrations/edit.js

```diff
@@ -17,4 +18,30 @@ $(() => {
     $('#org-controls .combobox-clear-button').hide();
     $('#other_org_toggle a').hide();
   }
+
+  // If the user changes their email address display the password
+  // confirmation modal
+  $('#personal_details_registration_form #user_email').blur((e) => {
```

Comment:
using blur here may break usability principles. I would be more inclined to add another handler for submit, apart from the one we have for validate the fields... and trigger the popup there instead.

Reply:
yes good point. will refactor to fire on submit if the password hasn't been confirmed